### PR TITLE
fix: Do not show left-hand sidebar for contributing page

### PR DESCRIPTION
**Pull request by glemaitre**
I assume that the contributing page has been renamed but the `conf.py` file not changed.
This PR should remove the useless left-hand sidebar.

### CODE REVIEW

**Review by MarieS-WiMLDS — CHANGES_REQUESTED** [review]
When I test on this branch, I still have the section navigation on the left on the contributing page:  
Capture d’écran du 2024-12-16 12-21-15

File: sphinx/conf.py [hunk 1]
```diff
@@ -139,7 +139,7 @@
     "install": [],
     "getting_started": [],
     "user_guide": [],
-    "contributor_guide": [],
+    "contributor_link": [],
```
Comment (MarieS-WiMLDS) [inline sphinx/conf.py:142]:
```suggestion
    "contributing_link": [],
```

Comment (glemaitre) [reply sphinx/conf.py:142]: Nice. I missed it :)

**Review by MarieS-WiMLDS — APPROVED** [review]

**Comment by sylvaincom** [comment]
Indeed, thanks!

**Pull request merged**